src/core/utils/KeyMapping.ts

```diff
@@ -0,0 +1,62 @@
+import { defaultKeyMap } from './DefaultKeyMap';
+
+export interface KeyModifiers {
+    ctrlKey?: boolean;
+    altKey?: boolean;
+    shiftKey?: boolean;
+}
+export interface Keypress {
+    key: string;
+    modifiers: KeyModifiers;
+}
+export interface IntentInformation {
+    name: string;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    arguments: Array<any> | Record<string, any>;
+}
+export type ShortcutIdentifier = string; // format: /[ctrl+][alt+][shift+]key/
```

Comment:
Oh, the type alias I referred to earlier in this review actually exists ! :D

Reply:
Ah ! There it was!